releasetool/commands/start/java.py

```diff
@@ -0,0 +1,229 @@
+        ctx.last_release_version = "0.0.0"
+
+    click.secho(f"The last release was {ctx.last_release_version}.")
+
+
+def determine_snapshot_version(ctx: Context) -> None:
+    click.secho("> Figuring out the current snapshot version.", fg="cyan")
+
+    with open("pom.xml", "r") as fh:
+        content = fh.read()
+        m = re.search(r'<version>(\d+\.\d+\.\d+)-SNAPSHOT</version>', content)
+        if m:
+            ctx.snapshot_version = m.group(1)
+
+    if ctx.snapshot_version is None:
+        click.secho(
+            "I couldn't figure out the current snapshot version from pom.xml.",
+            fg="red",
+        )
+        click.exit(1)
+
+
+def gather_changes(ctx: Context) -> None:
+    click.secho(f"> Gathering changes since {ctx.last_release_version}", fg="cyan")
+    ctx.changes = releasetool.git.summary_log(ctx.last_release_committish)
+    click.secho(f"Cool, {len(ctx.changes)} changes found.")
+
+
+def edit_release_notes(ctx: Context) -> None:
+    click.secho(f"> Opening your editor to finalize release notes.", fg="cyan")
+    release_notes = "\n".join(f"- {change}" for change in ctx.changes)
+    release_notes += "\n\n### ".join(
+        [
+            "",
+            "Implementation Changes",
+            "New Features",
+            "Dependencies",
+            "Documentation",
+            "Internal / Testing Changes",
+        ]
+    )
+    ctx.release_notes = releasetool.filehelpers.open_editor_with_tempfile(
+        release_notes, "release-notes.md"
+    ).strip()
+
+
+def determine_release_version(ctx: Context) -> None:
+    click.secho(f"> Now it's time to pick a release version!", fg="cyan")
+    release_notes = textwrap.indent(ctx.release_notes, "\t")
+    click.secho(f"Here's the release notes you wrote:\n\n{release_notes}\n")
+
+    parsed_version = [int(x) for x in ctx.last_release_version.split(".")]
+
+    if parsed_version == [0, 0, 0]:
+        ctx.release_version = "0.1.0"
+        return
+
+    selection = click.prompt(
+        "Is this a major, minor, or patch update (or enter the new version " "directly)"
+    )
+    if selection == "major":
+        parsed_version[0] += 1
+        parsed_version[1] = 0
+        parsed_version[2] = 0
+    elif selection == "minor":
+        parsed_version[1] += 1
+        parsed_version[2] = 0
+    elif selection == "patch":
+        parsed_version[2] += 1
+    else:
+        ctx.release_version = selection
+        return
+
+    ctx.release_version = "{}.{}.{}".format(*parsed_version)
+
+    click.secho(f"Got it, releasing {ctx.release_version}.")
+
+
+def create_release_branch(ctx) -> None:
+    ctx.release_branch = f"release-{ctx.package_name}-v{ctx.release_version}"
+    click.secho(f"> Creating branch {ctx.release_branch}", fg="cyan")
+    return releasetool.git.checkout_create_branch(ctx.release_branch)
+
+
+def gather_pom_xml_files(ctx: Context) -> None:
+    ctx.pom_files = glob('**/pom.xml', recursive=True)
+    print(ctx.pom_files)
```

Comment:
plain print? If it's for debugging, please remove, otherwise, please use click.secho.

Reply:
removed.